### PR TITLE
add XML examples

### PR DESCRIPTION
**Pull request by sjwang90**
added XML examples:
- cdc COVID-19 vaccine HTTP/XML config 
- London bikeshare XML file
- London bikeshare file/XML config

### CODE REVIEW

**Review by Jagularr — APPROVED** [review]

File: plugins/parsers/xml/testcases/cdc-vaccines.conf [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        metric_name = "'cdc-vaccines'"
+        timestamp = "week_of_allocations"
+        timestamp_format = "2006-01-02T15:04:05"
+
+        [inputs.http.xml.tags]
+            state   = "jurisdiction"
+
+        [inputs.http.xml.fields_int]
+            1st_dose_allocations = "_1st_dose_allocations"
+            2nd_dose_allocations = "_2nd_dose_allocations"
```
Comment (srebhan) [inline plugins/parsers/xml/testcases/cdc-vaccines.conf:34]: Not completely sure, but shouldn't this have the same indention level as `data_format` and `tagexclude`?

File: plugins/parsers/xml/testcases/cyclehire-london.conf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    field_selection = "child::info"
+    field_name = "name(@*[1])"
+    field_value = "number(@*[1])"
+
+    [inputs.tail.xml.tags]
+      address = "@name"
+      id = "@id"
+
+    [inputs.tail.xml.fields]
+      placement = "string(temporary)"
```
Comment (srebhan) [inline plugins/parsers/xml/testcases/cyclehire-london.conf:34]: The test-cases (unfortunately) have another format. You should do:
```suggestion

metric_selection = "response/child::location"
metric_name = "string('bikes')"

timestamp = "/stations/@lastUpdate"
timestamp_format = "unix_ms"

field_selection = "child::info"
field_name = "name(@*[1])"
field_value = "number(@*[1])"

[tags]
  address = "@name"
  id = "@id"

[fields]
  placement = "string(temporary)"
```

**Review by srebhan — CHANGES_REQUESTED** [review]
Hey Samantha, the test-case format is only a simplified version of the telegraf config and _only_ contains the parser's part without prefixes and can only read local files. Please take a look at my comment regarding the second test-case.

Furthermore, I fear that, due to these limitations, you cannot represent your first test case as is. You might download one of those XML files and put it locally and then parse it without the `enum` processor to generate a testable test-case.

Additionally or alternatively, we might want to generate an _examples_ subdir in the `xml` parser where you can put those full configs and then add a link to the parser's readme. What do you think?

**Comment by sjwang90** [comment]
As discussed on slack, instead of creating a separate `examples` folder we can just add them to our customer facing docs - docs.influxdata.com

**Pull request closed without merging**